sdk/storage/azure_storage_blob/src/clients/blob_service_client.rs

```diff
@@ -126,7 +126,7 @@ impl BlobServiceClient {
     /// * `url` - The full URL of the Azure storage account, for example `https://myaccount.blob.core.windows.net/`.
     /// * `credential` - An optional implementation of [`TokenCredential`] that can provide an Entra ID token to use when authenticating.
     /// * `options` - Optional configuration for the client.
-    pub fn from_url(
+    pub fn from_service_url(
```

Comment:
So, from a public API perspective, we don't actually need to offer a `from_url` type method since the URL is just the endpoint on a service client. `new` can be used in that case. We can remove this method entirely.